### PR TITLE
Clean up FreePlayerGFX

### PR DESCRIPTION
**Pull request by AJenbo**
Bin exact (probably already was), but debug from beta is pretty clear about mem_free_dbg only having one line between them, however I was unable to find a way to do this and still get bin exact.

### CODE REVIEW

File: Source/player.cpp [hunk 1]
```diff
@@ -452,50 +452,40 @@ int __fastcall GetPlrGFXSize(char *szCel)
...
-	plr[v2]._pDData = 0;
-	mem_free_dbg(v10);
-	v11 = plr[v2]._pBData;
-	plr[v2]._pBData = 0;
-	mem_free_dbg(v11);
-	plr[v2]._pGFXLoad = 0;
+	}
+
+	ptr = plr[pnum]._pNData;
+	plr[pnum]._pNData = 0;
```
Comment (seritools) [inline Source/player.cpp]: replace the 0 with `NULL` for pointers

**Comment by galaxyhaxz** [comment]
It was probably done using a macro, I can almost guarantee. Some like:
```
#define MemFreeDbg(x) x = NULL; \
                      mem_free_debug(x);
```

**Comment by seritools** [comment]
That would try to free NULL, wouldn't it? :D

**Comment by nomdenom** [comment]
Could also be an inline function. As a macro that would generally be written as:
```c
#define MemFreeDbg(ptr) do { \
  void* temp = ptr; \
  ptr = NULL; \
  mem_free_debug(temp); \
} while(0);
```

**Pull request merged**